prisma/seed.ts

```
@@ -35,6 +35,53 @@ async function main() {
+  const events = await Promise.all(
+    Array.from({ length: 10 }).map(async (_, index) => {
+      const startDate = new Date();
+      const endDate = new Date(startDate);
+      endDate.setHours(startDate.getHours() + 4);
+
+      // Generate only for even events
+      const includeImages = index % 2 === 0;
+      const imageIds: string[] = [];
+
+      if (includeImages) {
+        const images = await Promise.all(
```

ESLint

[error] 'images' is assigned a value but never used.

(no-unused-vars)